src/main/scala/temple/ast/Templefile.scala

```diff
@@ -19,7 +20,24 @@ case class Templefile(
   def servicesWithPorts: Iterable[(String, ServiceBlock, Ports)] =
     services
       .zip(Iterator.from(1025, step = 2)) // 1024 is reserved for auth service
-      .map { case ((name, service), port) => (name, service, Ports(port, port + 1)) }
+      .map {
+        case ((name, service), port) =>
+          if (name == "Auth") (name, service, Ports(ProjectConfig.authPort, ProjectConfig.authPort + 1))
+          else (name, service, Ports(port, port + 1))
+      }
+
+  def servicesWithPortsWithoutAuth: Iterable[(String, ServiceBlock, Ports)] =
```

Comment:
How about `providedServicesWithPorts` and `allServicesWithPorts` - I think without sounds a bit verbose?